modules/microshift-greenboot-testing-workload-script.adoc

```diff
@@ -34,18 +39,47 @@ $ sudo journalctl -o cat -u greenboot-healthcheck.service
 {microshift-short} core service health checks run before the workload health checks.
 ====
 +
-.Example output
+.Example output for a RHEL for Edge system
+[source,terminal]
+----
+GRUB boot variables:
+boot_success=0
+boot_indeterminate=0
+Greenboot variables:
+GREENBOOT_WATCHDOG_CHECK_ENABLED=true
+MICROSHIFT_WAIT_TIMEOUT_SEC=600
+System installation type:
+ostree
+System installation status:
+* rhel 19619bd269094510180c845c44d0944fd9aa15925376f249c4d680a3355e51ae.0
+    Version: 9.4
+    origin refspec: edge:rhel-9.4-microshift-4.18
+----
++
+.Example output for an image mode for RHEL system
```

Comment:
Is image mode the same as bootc in this case ?

My output differs from your example

Reply:
Yes, `Image Mode` is an official name for `bootc`

Reply:
What are you seeing on a bootc system?

Reply:
From running the following command

`sudo journalctl -o cat -u greenboot-healthcheck.service`

```
Running Required Health Check Scripts...
STARTED
GRUB boot variables:
boot_success=0
Greenboot variables:
GREENBOOT_WATCHDOG_CHECK_ENABLED=true
The ostree status:
* default 25e6c9b9d4921afc7ff39ce812a1cfae331e447d57a0b0847774fee12b3af26e.0
    origin: <unknown origin type>
Waiting 300s for MicroShift service to be active and not failed
Waiting 300s for MicroShift API health endpoints to be OK
```
Maybe I am running the wrong command ?